data/categories.yml

```diff
@@ -0,0 +1,216 @@
+- title: Economic Abuse Defined in State Laws
```

Comment:
@sarahgreen This is the file that imports all the information about the scorecard itself. We shouldn't have to modify this unless the FreeFrom team wants to make changes to the scorecard itself.